readthedocs/projects/management/commands/dump_project_remote_repo_relation.py

```diff
@@ -0,0 +1,45 @@
+import json
+
+from django.core.management.base import BaseCommand
+
+from readthedocs.projects.models import Project
+
+
+class Command(BaseCommand):
+    help = "Dump Project and RemoteRepository Relationship in JSON format"
+
+    def handle(self, *args, **options):
+        data = []
+
+        queryset = Project.objects.filter(
+            remote_repository__isnull=False,
+        ).values_list('id', 'remote_repository__json').distinct()
```

Comment:
We are not able to use the json lookup to just get the `remote_id` in the `values_list()` as we are currently using `TextField` to store the json data. Also, by getting the whole field we have more flexibility to check for inconsistent data at the dump stage so that we have clean and consistent data in the load stage.